### PR TITLE
EG-2976 Changes to TLS keys in HSM section as per feedback

### PR DESCRIPTION
**Pull request by ivanterziev-r3**
EG-2976 Changes to TLS keys in HSM section as per feedback

### CODE REVIEW

File: content/en/docs/corda-enterprise/4.6/ha-utilities.md [hunk 1]
```diff
@@ -261,6 +261,13 @@ Before re-running the tool, previously created keystore files must be removed fr
 
 {{< /note >}}
 
+The HA Utilities tool produces the following files:
```
Comment (rekalov) [inline content/en/docs/corda-enterprise/4.6/ha-utilities.md:264]: Can we correct to "The command produces ..." or "The following files are produced:" ?

File: content/en/docs/corda-enterprise/4.6/node/setup/tls-keys-in-hsm.md [hunk 2]
```diff
@@ -58,128 +58,6 @@ A file-based keystore is still required to store TLS certificates, even if corre
...
-enterpriseConfiguration = {
-    tlsCryptoServiceConfig = {
-       cryptoServiceName="AWS_CLOUD"
-       cryptoServiceConf="aws_cloud.conf"
-    }
-}
-```
-{{% /tab %}}
-{{< /tabs >}}
-
```
Comment (rekalov) [inline content/en/docs/corda-enterprise/4.6/node/setup/tls-keys-in-hsm.md:106]: Can we leave this section? Just as "Configuration example" (no need to mention all-in-one).

File: content/en/docs/corda-enterprise/4.6/ha-utilities.md (hunk 1 above)
Comment (ivanterziev-r3) [reply content/en/docs/corda-enterprise/4.6/ha-utilities.md:264]: Yes, also saw a few other things (legacy content issues) that I am fixing.

File: content/en/docs/corda-enterprise/4.6/node/setup/tls-keys-in-hsm.md (hunk 2 above)
Comment (ivanterziev-r3) [reply content/en/docs/corda-enterprise/4.6/node/setup/tls-keys-in-hsm.md:106]: Done

File: content/en/docs/corda-enterprise/4.6/node/setup/tls-keys-in-hsm.md [hunk 3]
```diff
@@ -47,47 +47,9 @@ The HA Utilities tool produces the following files:
 
 ### Storing TLS certificates in HSM
 
-A file-based keystore is still required to store TLS certificates, even if corresponding TLS keys are stored in CryptoService.
+A file-based `sslkeystore.jks` is still required to store TLS certificates, even if corresponding TLS keys are stored in CryptoService.
```
Comment (rekalov) [inline content/en/docs/corda-enterprise/4.6/node/setup/tls-keys-in-hsm.md:50]: Can you change "Storing TLS certificates in HSM" to "Storing TLS certificates" (or even remove this header and move sentence to the end "Configuration fields in node.conf")?

Also, we need to delete "Related configuration options" and "HA Utilities and self-signed internal Artemis SSL keystore"

**Review by edward-prosser — DISMISSED** [review]
I think those look alright

File: content/en/docs/corda-enterprise/4.6/node/setup/tls-keys-in-hsm.md (hunk 3 above)
Comment (ivanterziev-r3) [reply content/en/docs/corda-enterprise/4.6/node/setup/tls-keys-in-hsm.md:28]: Moved, and yes - sorry - I had forgotten to delete those.

**Review by rekalov — DISMISSED** [review]
Looks good.

File: content/en/docs/corda-enterprise/4.6/node/setup/tls-keys-in-hsm.md [hunk 4]
```diff
@@ -104,83 +44,6 @@ enterpriseConfiguration = {
 {{% /tab %}}
 {{< /tabs >}}
 
-### External bridge + embedded Artemis: simple configuration without HSM
-
-`node.conf`:
-
-{{< tabs name="tabs-2" >}}
-{{% tab name="kotlin" %}}
```
Comment (rekalov) [inline content/en/docs/corda-enterprise/4.6/node/setup/tls-keys-in-hsm.md:112]: actually one last thing (hopefully).
node.conf is not kotlin, so if you can mark it somehow else, please change

Comment (ivanterziev-r3) [reply content/en/docs/corda-enterprise/4.6/node/setup/tls-keys-in-hsm.md:112]: done

**Review by rekalov — DISMISSED** [review]

File: content/en/docs/corda-enterprise/4.6/node/setup/corda-configuration-fields.md [hunk 5]
```diff
@@ -254,22 +254,22 @@ Allows fine-grained controls of various features only available in the enterpris
...
+  * FAIL_FAST (no initial attempts, no reconnect attempts), CONTINUOUS_RETRY (infinite initial and reconnect attempts, starting retry interval of 5 seconds with an exponential back-of multiplier of 1.5 up for up to 5 minutes retry interval).
+  * *Default:* DEFAULT
+* `messagingServerBackupAddresses`
+  * List of Artemis Server back-up addresses. If any back-ups are specified, the client will be configured to automatically failover to the first server it can connect to.
+  * *Default:* empty list
+* `artemisCryptoServiceConfig`
+  * This is an optional crypto service configuration which will be used for HSM TLS signing when interacting with the Artemis message server.
+  * This option only makes sense when `messagingServerSslConfiguration` is specified: either to connect to a standalone Artemis messaging server, or when external Bridge is configured. If this option is missing, the local file system will be used to store private keys inside JKS key stores, as defined by `messagingServerSslConfiguration`.
+  * If this option is missing, the local file system will be used to store private keys inside `JKS` key stores.
```
Comment (taylorharvey-r3) [inline content/en/docs/corda-enterprise/4.6/node/setup/corda-configuration-fields.md:267]: Is this an almost repeat of the bullet point directly above?

**Review by taylorharvey-r3 — DISMISSED** [review]
One comment on a line that may be repeated. Everything else LGTM.

File: content/en/docs/corda-enterprise/4.6/node/setup/corda-configuration-fields.md (hunk 5 above)
Comment (ivanterziev-r3) [inline content/en/docs/corda-enterprise/4.6/node/setup/corda-configuration-fields.md:267]:
```suggestion
```

Comment (ivanterziev-r3) [reply content/en/docs/corda-enterprise/4.6/node/setup/corda-configuration-fields.md:267]: good spot - removing (this is due to 3 consecutive edits to this part from 3 different PRs)

**Review by taylorharvey-r3 — APPROVED** [review]
LGTM

**Pull request merged**